.github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+## Contributing
+
+As a contributor please ensure you have read over and understand Puppets
+Contributing guidelines, they can be found in the root directory of all Supported Modules.
```

Comment:
Following up on previous conversations (and the link at the very bottom) should we not move most of the CONTRIBUTING.md content to `iac/docs` and link to there to get everyone on the same page?